### ISSUE TITLE
POSIX host module

### ISSUE THREAD

**Issue by LuisHsu** [WasmVM/WasmVM#264]
Survey POSIX to determine a host module for later libc support

**Comment by Jack-Lin626** [comment]
So our host module aims to support the OS with POSIX. In other words, we can make sure our side can access local machine's resources as long as we follow POSIX?

**Comment by LuisHsu** [comment]
Exactly, this is just a rough idea, and it may also need some other works.
